### PR TITLE
LRDOCS-8909 Be sure we configure Advanced File System Store

### PR DESCRIPTION
**Pull request by achaparro**

### CODE REVIEW

File: docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay/upgrade-basics/using-the-database-upgrade-tool.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ If you're [upgrading to a new Liferay Docker image](../../installing-liferay/usi
 
 1. Set up the [File Store (Document Library)](../../../system-administration/file-storage/configuring-file-storage.md) by copying it from your [backup](../../maintaining-a-liferay-dxp-installation/backing-up.md) to the new installation and or configuring the new installation to use it via a [`.config` file](../../../system-administration/configuring-liferay/understanding-configuration-scope.md).
```
Comment (achaparro) [inline docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay/upgrade-basics/using-the-database-upgrade-tool.md:23]: change the link to https://help.liferay.com/hc/en-us/articles/360017877292-Understanding-System-Configuration-Files

Mention export/import when using AdvancedFileSystemStore

File: docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay/upgrade-basics/upgrading-via-docker.md [hunk 2]
```diff
@@ -45,15 +45,15 @@ Here are the steps for upgrading with a Docker image:
...
+    * `/osgi/configs/*.config`: [OSGi configuration files](../../../system-administration/configuring-liferay/understanding-configuration-scope.md).
 
     * `portal-*.properties`: [Portal properties](../../reference/portal-properties.md) files, such as `portal-ext.properties`.
 
     * `setenv.sh`, `startup.sh`, and more: Application server configuration scripts.
 
     * `web.xml`: Portal web application descriptor.
 
-1. Configure the [File Store (Document Library)](../../../system-administration/file-storage/configuring-file-storage.md) by exporting it's settings to a [`.config` file](../../../system-administration/configuring-liferay/understanding-configuration-scope.md) to use in your `new-version/osgi` folder.
+1. Configure the [File Store (Document Library)](../../../system-administration/file-storage/configuring-file-storage.md) by exporting it's settings to a [`.config` file](../../../system-administration/configuring-liferay/understanding-configuration-scope.md) to use in your `new-version/osgi` folder. This is only required when using Advanced File System Store or when you modified the storage location.
```
Comment (achaparro) [inline docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay/upgrade-basics/upgrading-via-docker.md:56]: Change the link too

**Comment by jhinkey** [comment]
Working on this now. Thanks!

**Comment by jhinkey** [comment]
sent https://github.com/achaparro/liferay-learn/pull/8

**Pull request closed without merging**